CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - [#5939](https://github.com/influxdata/influxdb/issues/5939): Support viewing and killing running queries.
 - [#6073](https://github.com/influxdata/influxdb/pulls/6073): Iterator stats
 - [#6079](https://github.com/influxdata/influxdb/issues/6079): Limit the maximum number of concurrent queries.
+- [#6075](https://github.com/influxdata/influxdb/issues/6075): Limit the maximum running time of a query.
```

Comment:
Shouldn't this be `[#6081](https://github.com/influxdata/influxdb/pull/6081)` ? Looks like some previous ones in that list aren't quite right either.